client/question/components/LearnerViewDirective.js

```diff
@@ -21,7 +21,7 @@ tie.directive('learnerView', [function() {
     restrict: 'E',
     scope: {},
     template: `
-      <div class="tie-wrapper" ng-class="{'night-mode': isInDarkMode}">
+      <div class="tie-wrapper" ng-class="{'night-mode': isInDarkMode()}">
```

Comment:
why not have this be IsDarkModeEnabled or DarkModeEnabled? IsInDarkMode seems like an odd construction, to me, since it's a setting.

Reply:
Done. This was a holdover from before, I agree that your suggestion is better. Thanks!